### ISSUE TITLE
fails to build on ubuntu 14.04 (travis)

### ISSUE THREAD

**Issue by lunixbochs** [keystone-engine/keystone#187]
I noticed your travis build is osx. I get this when building on linux travis:

> Host compiler appears to require libatomic, but cannot find it.

Does MC really depend on atomic operations, or is that a different part of LLVM we can skip?

This and #186 are blocking me merging Usercorn's keystone support.

**Comment by lunixbochs** [comment]
oh, I forgot travis was on 12.04, which doesn't have gcc 4.8. can probably work around this by switching to the trusty worker.

**Comment by aquynh** [comment]
good question, i saw this but havent looked into that yet. no idea about that atomic operations. would be great if you can make this work on Ubuntu 12.04

**Comment by lunixbochs** [comment]
yeah, works on trusty https://travis-ci.org/lunixbochs/usercorn/builds/135772914 so I just need the Go fix and Usercorn will have keystone :)

**Comment by aquynh** [comment]
great, please send a PR for Travis setup for Linux support.

**Comment by lunixbochs** [comment]
dunno, can't get it to honor distro *and* build for osx
